recipe/meta.yaml

```diff
@@ -82,7 +82,8 @@ outputs:
         - python
         - {{ pin_subpackage('pytensor-base', exact=True) }}
         - {{ compiler('c') }}  # [not win]
-        - cxx-compiler  # [not win]
+        - {{ compiler('cxx') }}  # [not win]
+        - gxx  # [linux]
```

Comment:
Should not be needed, `{{ compiler('cxx') }}` should be enough here.

Reply:
@ocefpaf, you're definitely a conda-forge superstar.

I thought that `{{ compiler('cxx') }}` was enough, but for whatever reason it's not. The `index.json` in the corresponding artifact has the following c-related packages:

```
gcc_linux-64 12.*
gxx_linux-64 12.*
libgcc-ng >=12
libstdcxx-ng >=12
```

To check, I do `docker run --rm -it mambaorg/micromamba` and run
```
micromamba install -c conda-forge "gcc_linux-64 12.*" "gxx_linux-64 12.*" "libgcc-ng >=12" "libstdcxx-ng >=12"
```
after which `g++` isn't found. But then installing `gxx` actually gives me the `g++` command.

Could it be since it's a runtime dependency?

Reply:
Probably. @isuruf can confirm that for us.

Reply:
I also [posted to Matrix](https://matrix.to/#/!SOyumkgPRWoXfQYIFH:matrix.org/$168875777680598qaQOI:matrix.org?via=matrix.org&via=gitter.im&via=cadair.com).